README.md

```diff
@@ -1044,6 +1044,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Isha Mehta](https://github.com/isha614)
   - [Ishan Kotian](https://github.com/Ishan-Kotian)
   - [Ishaan Choubey](https://github.com/1sh22)
+  - [Ishan Velle](https://github.com/foss2cyber)
```

Comment:
I believe your name must be one line above due to the extra `a`.
Let us know if you need any help doing that. 🙂

Reply:
Oh, I see. Thank you for the suggestion, mate! :slightly_smiling_face: I'll make the change right away.